leaflet/templates/leaflet/admin/change_list_geo.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'admin/change_list.html' %}
+{% load leaflet_tags %}
+{% load admin_list_leaflet %}
```

Comment:
Hi,
This file  "admin_list_leaflet" haven’t in the commit and templatetags. Where does it come from?  I am interested in this feature, it’s a good idea.
Thanks